_projects/git_task_list.html

```diff
@@ -0,0 +1,1137 @@
+<!DOCTYPE html><html><head><meta charset="utf-8"><meta name="collaborating_projects" content=""><style>body {
```

Comment:
wtf is this :P 
this commit is 30 mins old in the review queue, and you havent fixed it.